invenio_app_ils/indexer.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Ils Records API."""
+
+from __future__ import absolute_import, print_function
+
+import logging
```

Comment:
I think there is an invenio-logging or something like that for the logs, I guess these are left overs from development, also line:47